### PR TITLE
[cdoc] hadoop ai

### PR DESCRIPTION
**Pull request by YitongFeng**

### CODE REVIEW

File: hadoop-ai/README-zh.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+我们给 Hadoop 增加了 GPU 支持，增强其对 AI job 的调度性能。
+目前，YARN-3926 也将 GPU 作为可数资源（Countable Resource），支持 GPU 调度。
+然而，深度学习任务中，使用不同位置组合的 GPU 将影响 job 的效率。例如，若 GPU 0 和 1 同属于一个 PCI-E 交换机而 GPU 0 和 7 不是，则一个 2-GPU job 使用 GPU {0, 1}，将比使用 GPU {0, 7} 速度更快，
+
+我们给 Hadoop 2.7.2 添加了 GPU 支持以实现细粒度的 GPU 位置调度。
+
+我们增加了一个 64 位的 Bit-map 作为 YARN 资源，该 Bit-map 能够表示每个节点的 GPU 使用率及位置信息。每个节点共 64 个 GPU 位置，当该位置有 GPU 时对应的 Bit-map 位为“1”，否则为“0”。
+
+该 Hadoop AI 增强包下载地址：
```
Comment (abuccts) [inline hadoop-ai/README-zh.md]: patch -> '补丁'?

File: hadoop-ai/README-zh.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        make -j $(nproc)
+        make check -j $(nproc)
+        sudo make install
+        sudo ldconfig
+
+
+ 2. 下载 Hadoop AI 增强包
+    在 "hadoop-2.7.2-gpu-port.patch" from https://issues.apache.org/jira/browse/YARN-7481 下载Hadoop AI 增强包。
+   
+ 3. 获取 Hadoop 2.7.2 原码
```
Comment (abuccts) [inline hadoop-ai/README-zh.md]: ’源码‘

File: hadoop-ai/README-zh.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        tar xzvf protobuf-2.5.0.tar.gz
+        cd protobuf-2.5.0
+        ./configure
+        make -j $(nproc)
+        make check -j $(nproc)
+        sudo make install
+        sudo ldconfig
+
+
+ 2. 下载 Hadoop AI 增强包
```
Comment (abuccts) [inline hadoop-ai/README-zh.md]: same

File: hadoop-ai/README-zh.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+   
+    编译官方 Hadoop 的详细步骤在此略过。
+    执行下个步骤前请确保以上步骤执行正确。    
+   
+5. 应用 Hadoop AI 增强包
+   拷贝下载的增强包至你的 Linux 编译 Hadoop 的根目录下，运行以下命令：
+  ```sh 
+   git apply hadoop-2.7.2.port-gpu.patch
+   ```
+    if you see the output below you have successfully applied this patch
```
Comment (abuccts) [inline hadoop-ai/README-zh.md]: need translation

Comment (YitongFeng) [reply hadoop-ai/README-zh.md]: solved

**Pull request merged**